src/module/rolls/roll.ts

```diff
@@ -79,25 +85,29 @@ export class IronswornRoll {
     this.preRollOptions = preRollOpts
     this.postRollOptions = postRollOpts
   }
-
-  static action(stat: string, value: number, adds?: number): IronswornRoll {
+  // TODO: a way to add arbitrary values for certain assets (Looper) and moves (Develop your Relationship)?
```

Comment:
cool - i'll add a comment to that effect.

knowing the intended use, though -- i think there's utility in this being able to set arbitrary number values unlinked to a stat/condition meter in place of a reference to a PC attribute. aside from the 'canonical' outliers in Looper and Develop Your Relationship, custom macros seem like exactly the sort of place where a user is more likely to need some offbeat stat.

reckon it'd work if the `stat` argument was typed as `number | string`? then if `typeof stat === 'string'`, it's presumed to be an attribute key. if `typeof stat === 'number'`, the value is used.

slightly further afield: what do you think of including an optional argument for a move ID? then the macro can include a move reference, if desired; the chief benefit from the user PoV is that it'd include the relevant move outcome text in the chat output. heck, this could even *override* a move's normal stat/progress roll options. this would provide one avenue for people to implement alternate stat rolls w/o necessarily editing the moves themselves.

Reply:
Hmm. Maybe we should just remove these helpers until we can think through what they're needed for. You're right, this one probably isn't as helpful as I had intended when I started.